Modules/Communications/html/live.html

```diff
@@ -91,8 +91,9 @@
                 var valid = true;
                 str.forEach(function(element){
                     dict[element] = document.getElementById("set"+element.toUpperCase()).value;
-                    if(parseFloat(dict[element])>1.0||parseFloat(dict[element])<-1.0){ 
-                        "values must be between -1 and 1".alert();
+                    var value = parseFloat(dict[element]);
+					if(value != 0 && value != 1.0 && value != -1.0){ 
```

Comment:
Can you indent those conventionally? Actually, can you just prettify this file? It looks like it was lazily formatted.

Reply:
Fine 😭

Reply:
But i am starting to fix shai's code, this won't end here

Reply:
Feel free. Youv'e got a green light. :traffic_light: